<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controle</class>
 <widget class="QDialog" name="controle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1525</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1531</width>
     <height>891</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="gifLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 35pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>controle technique</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrôle de l'éclairage et de la signalisation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrôle des freins </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>580</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrôle de la direction</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>650</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrôle de l'émission de polluants </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>presence de ceintures de securité</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>430</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>controle de securité :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>510</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>présence d'airbags</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>550</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>presence de systémes de retenue pour enfants</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox8">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>590</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Phares et feux</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox9">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>630</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Essuie-glaces et lave-glace</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox10">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>670</y>
      <width>83</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Klaxon</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox11">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>440</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>controle de la plaque d'immatriculation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox12">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>510</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>controle de rétroviseurs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox13">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>580</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>controle des  vitres</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox14">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>650</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>passer Les essais de conduite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>760</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
border-radius:10px;
font: 14pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>tester</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
